modules/channel-twilio/src/backend/client.ts

```diff
@@ -0,0 +1,213 @@
+import * as sdk from 'botpress/sdk'
+import LRUCache from 'lru-cache'
+import { Twilio, validateRequest } from 'twilio'
+
+import { Config } from '../config'
+
+import { Clients, MessageOption, TwilioRequestBody } from './typings'
+
+const MIDDLEWARE_NAME = 'twilio.sendMessage'
+
+export class TwilioClient {
```

Comment:
Can you add some minimal debug so we can troubleshoot if something goes wrong ? Try to implement it like channel-slack in client.ts (incoming/outgoing